contracts/deploy/mainnet/120_pool_booster_curve_arb.js

```diff
@@ -0,0 +1,126 @@
+const addresses = require("../../utils/addresses");
+const { deployOnArb } = require("../../utils/deploy-l2");
+const { encodeSaltForCreateX } = require("../../utils/deploy");
+
+const createxAbi = require("../../abi/createx.json");
+const PoolBoosterBytecode = require("../../artifacts/contracts/strategies/CurvePoolBooster.sol/CurvePoolBooster.json");
+const ProxyBytecode = require("../../artifacts/contracts/proxies/Proxies.sol/CurvePoolBoosterProxy.json");
+
+// --------------------------------!!! / WARNING \ !!!-----------------------------------------
+//
+// `encodedSalt`, ProxyBytecode and PoolBoosterBytecode should be EXACTLY the same as the 119 !!!
+// It is using createX to deploy contract at the SAME address as the one deployed in 119.
+//
+// --------------------------------------------------------------------------------------------
+
+module.exports = deployOnArb(
+  {
+    deployName: "120_pool_booster_curve",
+    forceDeploy: false,
+    //forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: false,
+    proposalId: "",
+  },
+  async ({ withConfirmation }) => {
+    const { deployerAddr } = await getNamedAccounts();
+    const sDeployer = await ethers.provider.getSigner(deployerAddr);
+    console.log(`Deployer address: ${deployerAddr}`);
+
+    const cCurvePoolBooster = await ethers.getContractAt(
+      "CurvePoolBooster",
+      addresses.zero
+    );
+    const cCurvePoolBoosterProxy = await ethers.getContractAt(
+      "CurvePoolBoosterProxy",
+      addresses.zero
+    );
+
+    console.log("\nStarting deployment using CreateX");
+    const rewardToken = addresses.mainnet.OUSDProxy;
```

Comment:
this the reward token on Arbitrum the mainnet OUSD contract? Or should it be the a token on the Arbitrum chain?
There is no bridged OUSD on Arbitrum.Only Wrapped OUSD (wOUSD) is bridged